.github/workflows/markdown-link-check.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Markdown links
+
+on:
+  pull_request:
+    paths:
+      - "**/*.md"
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install markdown-link-check
+        run: npm i -g markdown-link-check
+      - name: Run markdown-link-check on MD files
+        run: find . -name "*.md" | xargs -n 1 markdown-link-check -q
```

Comment:
```suggestion
        run: find . -name "*.md" | xargs -n 1 npx markdown-link-check -q
```

Reply:
Resolving this one too, since it is multiple invocations of the executable